moveit_core/robot_state/include/moveit/robot_state/robot_state.h

```diff
@@ -1178,9 +1132,8 @@ as the new values that correspond to the group */
      @param jump_threshold The struct holding jump thresholds to determine if a joint space jump has occurred.
      @return The fraction of the trajectory that passed.
   */
-  static double testJointSpaceJump(const JointModelGroup* group, std::vector<RobotStatePtr>& traj,
-                                   const JumpThreshold& jump_threshold);
-
+  double testJointSpaceJump(const JointModelGroup* group, std::vector<RobotStatePtr>& traj,
```

Comment:
i think we should add deprecated header for all these functions, unless it generates too many warnings in other parts of moveit...

https://github.com/ros-planning/moveit/blob/melodic-devel/moveit_core/distance_field/include/moveit/distance_field/distance_field.h#L195